DataAccess/Concrete/EntityFramework/Contexts/WebNetSampleDBContext.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using WebNetSample.Entity.Concrete;
+
+namespace DataAccess.Concrete.EntityFramework.Contexts
+{
+    public class WebNetSampleDBContext : DbContext
+    {
+        protected IConfiguration Configuration { get; set; }
+
+        public DbSet<Product> Products { get; set; }
+        public DbSet<Category> Categories { get; set; }
+        public DbSet<Supplier> Suppliers { get; set; }
+
+        public WebNetSampleDBContext()
+        {
+
+        }
+
+        public WebNetSampleDBContext(DbContextOptions dbContextOptions, IConfiguration configuration) : base(dbContextOptions)
+        {
+            Configuration = configuration;
+        }
+
+        //protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
```

Comment:
Actually, I have 2 computers, one is mac and the other one is windows so that I have 2 configurations because I couldn't be sure about if I'll use it again on my other computer or not. Therefore, I decided not to delete this. Should I still delete?